cmd/e2e/utilities_test.go

```diff
@@ -269,14 +270,21 @@ func allocate(buildID, sessionID string, cert tls.Certificate) error {
 	transport := &http.Transport{TLSClientConfig: tlsConfig}
 	client := &http.Client{Transport: transport}
 
+	//Grabs the port from the running service
+	svc := corev1.Service{}
+	if err := kubeClient.Get(ctx, types.NamespacedName{Namespace: testNamespace, Name: buildName}, &svc); err != nil {
```

Comment:
this it not accurate (and this is the reason that lots of end to end tests fail). The service is on "thundernetes-system" namespace and does not have the name of any GameServerBuild, but instead its called "thundernetes-controller-manager". 
Moreover, can we get the port only once, maybe before we start the tests? 
We can grab it in the suite_test.go file, store it in a local variable and then use it from all tests in the suite.